src/components/TimerControls.tsx

```diff
@@ -9,9 +12,15 @@ const TimerControls = ({
 }): JSX.Element => {
 	return (
 		<>
-			<button type="button" onClick={pauseTimer}>
-				{isTimerPaused ? "Resume" : "Pause"}
-			</button>
+			{isTimerPaused ? (
+				<PlayButton onClick={pauseTimer}>
+					<FaRegPlayCircle className="fa-icon" />
```

Comment:
Is there a reason for this className of `fa-icon`?

I dont see a reference to it being used anywhere else

Reply:
This can go.